packages/glimmer-runtime/tests/initial-render-test.ts

```diff
@@ -985,23 +985,23 @@ test("Case-sensitive tag has capitalization preserved", function() {
 
 let warnings = 0;
 
-const StyleAttribute = {
-  setAttribute(dom, element, attr, value) {
+class StyleAttributeManager extends AttributeManager {
+  setAttribute(dom, element, value) {
     warnings++;
-    AttributeChangeList.setAttribute(dom, element, attr, value);
-  },
+    super.setAttribute(dom, element, value);
+  }
   updateAttribute() {}
 };
```

Comment:
It doesn't really matter here, but in Ember this should be non-allocating, since it have the same `name` for all instances. (You could just make const STYLE_ATTRIBUTE_MANAGER = new Style... and keep returning the same instance for all tagNames)